src/components/ModelTableList/ModelTableList.js

```diff
@@ -271,38 +295,147 @@ function generateTableHeaders(label, count) {
   ];
 }
 
-function TableList() {
+/**
+  Generates the table headers owner grouped table
+  @param {String} label The title of the table.
+  @param {Number} count The number of elements in the status.
+  @returns {Array} The headers for the table.
+*/
+function generateOwnerTableHeaders(owner, count) {
+  return [
+    {
+      content: generateStatusIcon(owner, count, false),
+      sortKey: owner.toLowerCase()
+    },
+    { content: "Status", sortKey: "statusˇ" },
+    { content: "Configuration", sortKey: "summary" },
+    { content: "Cloud/Region", sortKey: "cloud" },
+    { content: "Credential", sortKey: "credential" },
+    { content: "Controller", sortKey: "controller" },
+    {
+      content: "Last Updated",
+      sortKey: "last-updated",
+      className: "u-align--right"
+    }
+  ];
+}
+
+function ModelTableList({ groupedBy }) {
   // Even though the activeUser tag is needed many functions deep, because
   // hooks _must_ be called in the same order every time we have to get it here
   // and pass it down through the `generateModelTableData` function as the order
   // of the model data changes frequently and it's guaranteed to almost never be
   // in the same order.
   const activeUser = useSelector(getActiveUserTag);
-  const groupedModelData = useSelector(getGroupedModelData);
-  const { blockedRows, alertRows, runningRows } = generateModelTableData(
-    groupedModelData,
-    activeUser
-  );
+  const groupedModelDataByStatus = useSelector(getGroupedModelDataByStatus);
+  const groupedModelDataByOwner = useSelector(getGroupedModelDataByOwner);
+  const {
+    blockedRows,
+    alertRows,
+    runningRows
+  } = generateModelTableDataByStatus(groupedModelDataByStatus, activeUser);
+  const ownerRows = generateModelTableDataByOwner(groupedModelDataByOwner);
 
-  return (
-    <>
-      <MainTable
-        className={"u-table-layout--auto"}
-        headers={generateTableHeaders("Blocked", blockedRows.length)}
-        rows={blockedRows}
-      />
-      <MainTable
-        className={"u-table-layout--auto"}
-        headers={generateTableHeaders("Alert", alertRows.length)}
-        rows={alertRows}
-      />
-      <MainTable
-        className={"u-table-layout--auto"}
-        headers={generateTableHeaders("Running", runningRows.length)}
-        rows={runningRows}
-      />
-    </>
-  );
+  switch (groupedBy) {
+    case "status":
+      return (
+        <>
```

Comment:
We should probably move each one of these 'groups' into its own sub component as this one is getting quite large.

This `ModelTableList` job would be to toggle between the various components based on the selection.